gestures.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: Gestures
+permalink: /gestures/
+---
+
+The gesture system in Flutter has two separate layers.  The first layer has raw
+pointer events, which describe the location and movement of pointers (e.g.,
+touches, mice, and styli) across the screen. The second layer has gestures,
+which describe semantic actions that consist of one or more pointer movements.
+
+# Pointers
+
+Pointers represent raw data about the user's interaction with the device's
+screen.  There are four types of pointer events:
+
+- `PointerDownEvent` The pointer has contacted the screen at a particular
+   location.
+- `PointerMoveEvent` The pointer has moved from one location on the screen to
+   another.
+- `PointerUpEvent` The pointer has stopped contacting the screen.
+- `PointerCancelEvent` The framework is no longer delivering events for this
```

Comment:
"The system has lost track of this pointer and will not generate more events for it." ?  What text do you recommend here?